src/config.ts

```diff
@@ -17,18 +17,22 @@ export class KubeConfig {
     // Only public for testing.
     public static findHomeDir(): string | null {
         if (process.env.HOME) {
-            if (fs.existsSync(process.env.HOME)) {
+            try {
+                fs.accessSync(process.env.HOME);
                 return process.env.HOME;
-            }
+            // tslint:disable-next-line:no-empty
+            } catch (ignore) {}
         }
         if (process.platform !== 'win32') {
             return null;
         }
         if (process.env.HOMEDRIVE && process.env.HOMEPATH) {
             const dir = path.join(process.env.HOMEDRIVE, process.env.HOMEPATH);
-            if (fs.existsSync(dir)) {
+            try {
+                fs.accessSync(dir);
                 return dir;
-            }
+            // tslint:disable-next-line:no-empty
+            } catch (ignore) {}
         }
         if (process.env.USERPROFILE && fs.existsSync(process.env.USERPROFILE)) {
```

Comment:
@brendanburns should we not replace this too? `fs.existsSync(process.env.USERPROFILE`

Reply:
done. (and a few other places I missed)